### DISCUSSION TITLE
why jaxlib 0.3.* is deleted from python package ?

### DISCUSSION THREAD

**Discussion by yudonglee** [jax-ml/jax#18367] [General] [answered]
My code failed to run today, and I found out It's caused by the unavailable jaxlib version. 
0.3.* is not too old, but as it's show int https://pypi.org/project/jaxlib/#history , 0.3.* version is not supported any more. why can't it be kept for a longer time ?

**Comment by sergiuiacob1** [comment]
+1 for this. This is the second / third time I've experienced CI pipelines failing suddenly over night due to jaxlib / jax / iree libraries having versions deprecated in the PyPi repository. Not to mention the amount of breaking changes that happen between version bumps, which means upgrading is not so straightforward.

This breaks long-living projects and should be treated more seriously. Specifically in my case, `jaxlib==0.3.15` is not available anymore.

**Comment by jakevdp** [comment, marked as answer]
See the response on the duplicate issue: https://github.com/google/jax/issues/18368#issuecomment-1792381092
